app/assets/stylesheets/application.scss

```diff
@@ -183,15 +195,70 @@ ul {
 }
 
 .recipe-body {
+  background-color: rgba(94, 94, 94, 0.7);
+  border-style: solid;
+  border-color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.